std-bits/table/src/main/java/org/enso/table/error/ColumnNameMismatchException.java

```diff
@@ -0,0 +1,26 @@
+package org.enso.table.error;
+
+import java.util.Arrays;
+
+public class ColumnNameMismatchException extends Exception {
+  private final String[] missing;
+  private final String[] extras;
+
+  public ColumnNameMismatchException(String[] missingNames, String[] extraNames) {
+    super(
+        "Columns mismatch."
+            + (missingNames.length == 0 ? "" : " Missing: " + Arrays.toString(missingNames))
+            + (extraNames.length == 0 ? "" : " Extra: " + Arrays.toString(extraNames)));
```

Comment:
Looking at this kind of error message, it is not immediately clear to me if 'missing' columns are ones that are present in _our table_ but are missing from the _file_ or vice-versa? I'm wondering if we can make the message make this more clear.